### PR TITLE
Feature/accentuate all time high and low

### PR DESCRIPTION
**Pull request by dtslvr**

### CODE REVIEW

**Review by vzickner — COMMENTED** [review]
In general, it looks good to me. But do we not still have the issue that if you don't have the timeline max that you might have a wrong all-time high?

**Comment by dtslvr** [comment]
> In general, it looks good to me. But do we not still have the issue that if you don't have the timeline max that you might have a wrong all-time high?

Yes, this is still open. But more critical is my unresolved comment above (`Error: [big.js] Invalid number`)

File: apps/api/src/app/portfolio/portfolio-calculator.ts [hunk 1]
```diff
@@ -417,11 +422,38 @@ export class PortfolioCalculator {
...
     }
 
-    const timelinePeriods: TimelinePeriod[][] = await Promise.all(
+    const timelineInfoInterfaces: TimelineInfoInterface[] = await Promise.all(
       timelinePeriodPromises
     );
+    const minNetPerformance = timelineInfoInterfaces
+      .map((timelineInfo) => timelineInfo.minNetPerformance)
+      .reduce((minPerformance, current) => {
+        if (minPerformance.lt(current)) {
```
Comment (dtslvr) [inline apps/api/src/app/portfolio/portfolio-calculator.ts:431]:
```
Error: [big.js] Invalid number
    at parse (/dev/ghostfolio/source/node_modules/big.js/big.js:142:13)
    at new Big (/dev/ghostfolio/source/node_modules/big.js/big.js:109:9)
    at Big.cmp (/dev/ghostfolio/source/node_modules/big.js/big.js:309:17)
    at Big.lt (/dev/ghostfolio/source/node_modules/big.js/big.js:479:17)
    at /dev/ghostfolio/source/dist/apps/api/webpack:/apps/api/src/app/portfolio/portfolio-calculator.ts:431:28
    at Array.reduce (<anonymous>)
    at PortfolioCalculator.<anonymous> (/dev/ghostfolio/source/dist/apps/api/webpack:/apps/api/src/app/portfolio/portfolio-calculator.ts:430:8)
    at Generator.next (<anonymous>)
    at fulfilled (/dev/ghostfolio/source/node_modules/tslib/tslib.js:111:62)
```

**Review by dtslvr — COMMENTED** [review]
Please see https://github.com/ghostfolio/ghostfolio/pull/428/files#r753839979

File: apps/api/src/app/portfolio/portfolio-calculator.ts (hunk 1 above)
Comment (dtslvr) [reply apps/api/src/app/portfolio/portfolio-calculator.ts:431]: This is problem is probably related:

```
TypeError: Cannot read property 'lt' of null
    at /dev/ghostfolio/source/dist/apps/api/webpack:/apps/api/src/app/portfolio/portfolio-calculator.ts:431:28
    at Array.reduce (<anonymous>)
    at PortfolioCalculator.<anonymous> (/dev/ghostfolio/source/dist/apps/api/webpack:/apps/api/src/app/portfolio/portfolio-calculator.ts:430:8)
    at Generator.next (<anonymous>)
    at fulfilled (/dev/ghostfolio/source/node_modules/tslib/tslib.js:111:62)
```

It happens shortly after `Missing value for symbol XYZ at 2021-04-14`.

**Pull request merged**